pkgs/development/python-modules/serverlessrepo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, boto3
+, pyyaml
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "severlessrepo";
+  version = "0.1.8";
+
+  src = fetchPypi {
+    pname = "serverlessrepo";
+    inherit version;
+    sha256 = "533389d41a51450e50cc01405ab766550170149c08e1c85b3a1559b0fab4cb25";
+  };
+
+  propagatedBuildInputs = [
+    boto3
+    pyyaml
+  ];
+
+  checkInputs = [ pytest mock ];
```

Comment:
Probably should run these with `pytest tests` in `checkPhase`.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/commit/0d1ec03d1a3e70a546183e53795398ecfdbfd881#diff-09d026033975cc29501d417399d1fd94R28